### PR TITLE
Explain MNO request statuses to school and RB users

### PR DESCRIPTION
**Pull request by benilovj**
### Context

Once an school or RB user makes an MNO request, the mobile network will either action it or provide feedback on it. It's not always obvious what each status means.

### Changes proposed in this pull request

Display a table of statuses and descriptions to school and RB users.

### Guidance to review

image

### CODE REVIEW

File: app/views/shared/internet/_help_with_statuses.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<%= govuk_details(summary: 'Help with statuses') do %>
+  <p class="govuk-body">View each request for guidance on what to do if a request fails.</p>
```
Comment (fofr) [inline app/views/shared/internet/_help_with_statuses.html.erb:2]: Drop this line for now

Comment (benilovj) [reply app/views/shared/internet/_help_with_statuses.html.erb]: Oops... fixed.

**Review by fofr — APPROVED** [review]
Design looks good :+1:

**Review by tonyheadford — APPROVED** [review]
LGTM!

**Pull request merged**